Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -331,6 +335,10 @@ private class TopSitesDataSource: NSObject, UICollectionViewDataSource {
     }
 
     subscript(index: Int) -> Site? {
+        if data.status != .Success {
+            return nil
+        }
```

Comment:
Is this necessary? If `numberOfItemsInSection` is returning 0 for this case, I wouldn't expect us to be trying to access any indices in the data.